frontend/src/components/ApolloAuthProvider.tsx

```diff
@@ -58,6 +58,19 @@ const ApolloAuthProvider: FunctionComponent = ({ children }) => {
             },
           },
         },
+        Pallet: {
+          fields: {
+            // This custom merge function is APPARENTLY necessary to avoid the
+            // following warning:
+            // "Cache data may be lost when replacing the lineItems field of a
+            // Pallet object"
+            lineItems: {
+              merge: (existing = [], incoming = []) => {
+                return [...incoming]
+              },
+            },
```

Comment:
suggestion (non-blocking): I usually wait until it'd be three to extract simple code duplication like this but with the comment maybe it's worth doing it now?

```ts
const mergeByReplacement = {
  merge: (existing = [], incoming = []) => {
    return [...incoming]
  },
}
```